exporter/chronicleexporter/factory.go

```diff
@@ -0,0 +1,83 @@
+// Copyright observIQ, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package chronicleexporter
+
+import (
+	"context"
+	"errors"
+
+	"github.com/observiq/bindplane-agent/exporter/chronicleexporter/internal/metadata"
+	"go.opentelemetry.io/collector/component"
+	"go.opentelemetry.io/collector/consumer/consumererror"
+	"go.opentelemetry.io/collector/exporter"
+	"go.opentelemetry.io/collector/exporter/exporterhelper"
+)
+
+// TypeStr for the exporter type.
+const TypeStr = "chronicle"
+
+// Factory is the factory for the Chronicle exporter.
+type Factory struct {
+}
```

Comment:
Are these used? I think `typeStr` comes from the metadata package now, and a factory struct should be unnecessary.

Reply:
Updated.